modules/tinymce/src/core/main/ts/newline/InsertBlock.ts

```diff
@@ -25,6 +25,11 @@ const trimZwsp = (fragment: DocumentFragment) => {
   });
 };
 
+const isEditableList = (editor: Editor, node: Node): boolean => {
+  const parentList = editor.dom.getParent(node, 'ol,ul,dl');
+  return !parentList ? true : editor.dom.getContentEditableParent(parentList) !== 'false';
```

Comment:
Using `closest` as the name for that would be better. I have always regretted that the `dom.getParent` is not called `dom.getClosest` since that is what it's really doing.